flutter/example/lib/main.dart

```diff
@@ -34,12 +34,20 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      navigatorObservers: [
+        SentryNavigatorObserver(),
+      ],
       home: Scaffold(
         appBar: AppBar(title: const Text('Sentry Flutter Example')),
         body: SingleChildScrollView(
           child: Column(
             children: [
               const Center(child: Text('Trigger an action:\n')),
+              RaisedButton(
+                child: const Text('Open another Scaffold'),
+                onPressed: () =>
+                    SecondaryScaffold.openSecondaryScaffold(context),
+              ),
```

Comment:
so not sure if there's a bug here or you did intentionally:

> Navigator operation requested with a context that does not include a Navigator.
> The context used to push or pop routes from the Navigator must be that of a widget that is a descendant of a Navigator widget.

Reply:
Oh shoot, you're right. That was definitely a mistake on my side. I've fixed it.

Reply:
cool, waiting for the CI #201 and you can merge main into your branch and GG :)